packages/grafana-data/src/transformations/transformers/reduce.ts

```diff
@@ -18,7 +18,7 @@ export interface ReduceTransformerOptions {
 export const reduceTransformer: DataTransformerInfo<ReduceTransformerOptions> = {
   id: DataTransformerID.reduce,
   name: 'Reduce',
-  description: 'Reduce all rows to a single row and concatenate all results',
+  description: 'Reduce all rows or data points to a single value using a function like max, min, mean or last',
```

Comment:
We currently have descriptions in the UI and in this package. A little bit confusing to me. any reason or just no time to decide where it should go?